.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = crlf
```

Comment:
I think the reason that .gitattributes is there at all is that people kept committing CRLF files and I had to clean them up. Would your proposed change cause git to clean them up on commit?